purchase_work_acceptance/views/work_acceptance_views.xml

```diff
@@ -0,0 +1,175 @@
+                           <field name="origin" attrs="{'invisible': [('origin','=',False)], 'readonly': [('state', '!=', 'draft')]}"/>
+                           <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
+                       </group>
+                   </group>
+                   <notebook>
+                       <page string="Products">
+                           <field name="wa_line_ids" attrs="{'readonly': [('state', '!=', 'draft')]}">
+                                <tree string="Work Acceptance Lines" editable="bottom">
+                                   <field name="currency_id" invisible="1"/>
+                                   <field name="state" invisible="1"/>
+                                   <field name="product_type" invisible="1"/>
+                                   <field name="product_id"/>
+                                   <field name="name"/>
+                                   <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                                   <field name="product_qty"/>
+                                   <field name="product_uom" groups="uom.group_uom" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
+                                   <field name="price_unit"/>
+                                   <field name="price_subtotal" widget="monetary" sum="Total"/>
+                               </tree>
+                               <form string="Work Acceptance Line">
+                                   <sheet>
+                                       <field name="state" invisible="1"/>
+                                       <group>
+                                           <group>
+                                               <field name="product_id"/>
+                                               <label for="product_qty"/>
+                                               <div>
+                                                   <field name="product_qty" class="oe_inline"/>
+                                                   <span class="oe_inline">&#160;</span>
+                                                   <field name="product_uom" groups="uom.group_uom" class="oe_inline"/>
+                                               </div>
+                                               <field name="price_unit"/>
+                                           </group>
+                                           <group>
+                                               <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                                           </group>
+                                       </group>
+                                   </sheet>
+                               </form>
+                           </field>
+                       </page>
```

Comment:
Please add Chartter. I think it will be very useful. When people do acceptance, they can have conversation back and forth.
Please also learn how PO is writing messages in the chatter. You can learn from it and make a nice history.